### ISSUE TITLE
alternatives to `bp.openssf.org`

### ISSUE THREAD

**Issue by ctcpip** [ossf/wg-best-practices-os-developers#162]
prefer to avoid an acronym, especially where overloaded and not common. is an alternative possible? e.g. `best.openssf.org` ?

mentioned in #161, but was off-topic

**Comment by david-a-wheeler** [comment]
@ctcpip - thanks for creating this new issue so we can keep track of things!

Domain names are often best short so it's easy to se what it is. The WG discussion today accepted "bp.openssf.org" so that's what I set up. "bp" is pretty short :-). Of course "best" is isn't very long either.

It would be easy to change though, and if we're going to change it, now is the time to change it.

We can easily make improvements to the CSS, etc., in parallel while we discuss the domain name.

**Comment by balteravishay** [comment]
+1 on best over bp, since bp is not a known acronym.

**Comment by ran-dall** [comment]
++1 on 'best' over 'bp'

**Comment by SecurityCRob** [comment]
+1 on "best"

**Comment by thomasnyman** [comment]
+1 on "best", there's a nice ring to it!

**Comment by oej** [comment]
I think "best.openssf.org" can be misunderstood as a selection of good stuff from OpenSSF. While it's a cool name to book in the openssf namespace, I personally prefer "bp" for best practices.

**Comment by 06kellyjac** [comment]
I'm inclined to go with `advice.openssf.org` but since the project is pretty committed to "Best Practices" I like either option

**Comment by david-a-wheeler** [comment]
NB: "bp.openssf.org" no longer works. The process for enabling "best.openssf.org" has begun. It'll take some time for this new name to work and propagate, but these are all one-time delays.

**Comment by david-a-wheeler** [comment]
Okay, it's running at best.openssf.org, with https: (TLS) enabled. Also: if you prefer, you can omit the trailing ".html". You can try it out here:

* https://best.openssf.org/Concise-Guide-for-Evaluating-Open-Source-Software
* https://best.openssf.org/Concise-Guide-for-Developing-More-Secure-Software

These pages are using the draft OpenSSF font recommendations (those can be changed if desired).
